src/80211_if.c

```diff
@@ -2941,6 +2974,13 @@ static int uccp420wlan_init(void)
 		}
 	}
 
+	error=start_core(hw);
```

Comment:
I got the confirmation from FW that they by default enable BT Coex after FW loading, so we can skip the bt_info programming. Instead we can just move the load_fw to init.

This FW change will be part of next release which is planned soon (6.9).  We can keep the driver changes ready and leave PR open till that FW is submitted.